src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="logo" href="#Home">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <div class="space"></div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__item__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
      <div class="space"></div>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__item__link is-active" href="#Apple">Apple</a>
          </li>
          
          <li class="nav__item">
```
fix code style all over the page